modules/600-secret-copier/hooks/handler.go

```diff
@@ -159,9 +159,13 @@ func copierHandler(input *go_hook.HookInput, dc dependency.Container) error {
 	secretsDesired := make(map[string]*Secret)
 	for _, s := range secrets {
 		secret := s.(*Secret)
+
 		// Secrets that are not in namespace `default` are existing Secrets.
 		if secret.Namespace != v1.NamespaceDefault {
 			path := SecretPath(secret)
+			// remove annotations because desired secret doesn't have them
+			// annotations map have to be nil that secrets are equal
+			secret.Annotations = nil
```

Comment:
It is better to do in ApplyCopierSecretFilter.

Reply:
And i didn't get the comment.

Reply:
Nope, annotations are needed after apply function, we use some value from them. But below this code we have a DeepEqual function, which is comparing `desired` secret with `existing` secret and `desired` secret does not have annotations, so we have to remove them from `existing` to satisfy equaty